package.json

```diff
@@ -24,11 +24,13 @@
     "url": "https://github.com/StevenMaMei/protractor-workshop-2020/issues"
   },
   "homepage": "https://github.com/StevenMaMei/protractor-workshop-2020#readme",
-
   "devDependencies": {
     "protractor": "^7.0.0",
     "@types/jasminewd2": "^2.0.8",
     "@types/node": "^14.0.25",
     "typescript": "^3.9.7"
+  },
+  "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2"
```

Comment:
this dependency should be installed as dev dependency.

Reply:
Done